libraries/AP_Logger/LogFile.cpp

```diff
@@ -914,6 +914,24 @@ void AP_Logger::Write_VisualOdom(float time_delta, const Vector3f &angle_delta,
     WriteBlock(&pkt_visualodom, sizeof(log_VisualOdom));
 }
 
+// Write visual position sensor data.  x,y,z are in meters, angles are in degrees
+void AP_Logger::Write_VisualPosition(uint64_t remote_time_us, uint32_t time_ms, float x, float y, float z, float roll, float pitch, float yaw)
+{
+    const struct log_VisualPosition pkt_visualpos {
+        LOG_PACKET_HEADER_INIT(LOG_VISUALPOS_MSG),
```

Comment:
Documentation is missing for this.

With three time fields it's probably important to have it :-)

Reply:
Thanks, VISP log message description added in follow-up commit